_opensearch/query-dsl/geo-bounding-box.md

```diff
@@ -0,0 +1,226 @@
+      "value" : 1,
+      "relation" : "eq"
+    },
+    "max_score" : 1.0,
+    "hits" : [
+      {
+        "_index" : "testindex1",
+        "_id" : "1",
+        "_score" : 1.0,
+        "_source" : {
+          "point" : {
+            "lat" : 74.0,
+            "lon" : 40.71
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+The preceding response does not include the document with a geopoint of `"lat": 75.00, "lon": 28.00` because of the geopoint's limited [precision](#precision).
+{: .note}
+
+## Precision
+
+Geopoint coordinates are always rounded down at index time. At query time, the bounding box upper boundaries are rounded down, and the lower boundaries are rounded up. Therefore, the documents with geopoints that lie on the lower and left edges of the bounding box might not be included in the results due to rounding error. On the other hand, geopoints that lie on the upper and right edges of the bounding box might be included in the results even though they are outside the boundaries. The rounding error is less than 4.20 &times; 10<sup>&minus;8</sup> degrees for latitude and less than 8.39 &times; 10<sup>&minus;8</sup> degrees for longitude (around 1 cm). 
+
+## Specifying the bounding box
+
+You can specify the bounding box by providing any of the following combinations of its vertex coordinates:
+
+- `top_left` and `bottom_right`
+- `top_right` and `bottom_left`
+- `top`, `left`, `bottom`, and `right`
+
+The following example specifies the bounding box using the `top`, `left`, `bottom`, and `right` coordinates:
+
+```json
+GET testindex1/_search
+{
+  "query": {
+    "bool": {
+      "must": {
+        "match_all": {}
+      },
+      "filter": {
+        "geo_bounding_box": {
+          "point": {
+            "top": 75,
+            "left": 28,
+            "bottom": 73,
+            "right": 41            
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Request fields
+
+Geo-bounding box queries accept the following fields.
+
+Field | Data type | Description
+:--- | :--- | :--- 
+_name | String | The name of the filter. Optional.
+validation_method | String | The validation method. Valid values are `IGNORE_MALFORMED` (accept geopoints with invalid coordinates), `COERCE` (try to coerce  coordinates to valid values), and `STRICT` (return an error when coordinates are invalid). Default is `STRICT`.
+type | String | Specifies how to execute the filter. Valid values are `indexed` (index  the filter) and `memory` (execute the filter in memory). Default is `memory`.
+ignore_unmapped | Boolean | Specifies whether to ignore an unmapped field. If set to `true`, the query does not return any documents that have an unmapped field. If set to `false`, an exception is thrown when the field is unmapped. Default is `false`.
+
+## Accepted formats
+
+You can specify coordinates of the bounding box vertices in any [format]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point#formats) that geopoint accepts.  
```

Comment:
The end of this sentence makes it sound as though "geopoint" is a proper noun. My brain wants it to be "that accepts geopoints." Can we revise for clarity?

Reply:
Reworded.